packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -36,6 +36,23 @@ export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationL
    * @default - Health check configuration from container.
    */
   readonly healthCheck?: HealthCheck;
+
+  /**
+   * The minimum number of CPU units to reserve for the container.
+   *
+   * @default - No minimum CPU units reserved.
+   */
+  readonly containerCpu?: number;
+
+  /**
+   * The amount (in MiB) of memory to present to the container.
+   *
+   * If your container attempts to exceed the allocated memory, the container
+   * is terminated.
+   *
+   * @default - No memory limit.
+   */
+  readonly containerMemoryLimitMiB?: number;
```

Comment:
I am not sure, but I think the current implementation is good. Because putting it in `FargateServiceBaseProps` would be risky because it would affect all constructs that use all props that implement it.

Also, a property such as `healthCheck`, for example, could be included as a common item in the Base Construct, but right now it is only in the `ApplicationLoadedBalancedFargateServiceProps` (and `QueueProcessingFargateServiceProps`). From this point of view, it would make sense to include them only in the props.

(However, to be honest, I also think that cpu and memory are common properties for containers. But if it became necessary, it could be modified later.)

Reply:
Understood, thank you for the detailed explanation!